packages/mdc-ripple/_keyframes.scss

```diff
@@ -20,46 +20,33 @@
 
 @keyframes mdc-ripple-fg-radius-in {
   from {
-    transform: translate(0) scale(1);
```

Comment:
@BBosman good question. Typically, yes. However, this keyframe is only attached when `mdc-ripple-upgraded` is added to the ripple surface. Before we add that class, we check - within JS - to make sure custom properties are supported. Thus, fallbacks here aren't necessary.

...I'm going to add this explanation as a comment 😃